<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GexDbPlugin_Medtronic_Rdb_OptionsWidget_Base</class>
 <widget class="QWidget" name="GexDbPlugin_Medtronic_Rdb_OptionsWidget_Base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="labelTestsWithMultipleExecutions">
         <property name="text">
          <string>How to handle tests with multiple executions per run:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="buttonGroupMultiExecutionTests">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QRadioButton" name="buttonKeepFirstResult">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>237</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Keep first result</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="buttonKeepLastResult">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>237</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Keep last result</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="buttonKeepAllResults">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>237</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Keep all results (create artificial tests)</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer32">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>54</width>
         <height>15</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer33">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>73</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
